### PR TITLE
Machine concept for configurable wire bundling.

### PR DESCRIPTION
**Pull request by Exaxxion**
Adds a line of simple machines called Bundlers which join wires together into thicker wires.

It uses an Integrated Circuit to specify how many tiers above the current wire thickness to bundle incoming stacks of wires. The goal of this machine is to reduce strain on automation systems like AE2 molecular assemblers, instead offering a specialized machine that is also compatible with the Processing Array.

### CODE REVIEW

**Comment by Exaxxion** [comment]
Just rebasing commits to latest master.

**Comment by Exaxxion** [comment]
All right, I think this is usable as it is implemented currently.

It would be nice to have a different texture for these machines (currently they're just using the Packing Machine texture), but it does work fine as it is currently.

**Comment by DStrand1** [comment]
I want to preface this by saying that @ALongStringOfNumbers asked me to review this PR.

Implementation-wise it is solid, I see no issues. However, personally I don't see why this should be in its own machine. Can these recipes not simply be added to the Packing Machine to save some machine bloat? I am of the mind that GregTech already has a few too many machines (2 Extractors, 2 Canners, Compressor and Bending Machine, Circuit and Regular Assembler in some versions, etc.), so personally I don't like another one-off machine.

File: src/main/resources/assets/gtadditions/lang/en_us.lang [hunk 1]
```diff
@@ -705,6 +706,18 @@ gregtech.machine.assembler.iv.name=Assembler IV
 gregtech.machine.autoclave.iv.name=Autoclave IV
 gregtech.machine.laser_engraver.iv.name=Laser Engraver IV
 
+gtadditions.machine.bundler.lv.name=Basic Bundler
+gtadditions.machine.bundler.mv.name=Advanced Bundler
+gtadditions.machine.bundler.hv.name=Advanced Bundler II
+gtadditions.machine.bundler.ev.name=Advanced Bundler III
+gtadditions.machine.bundler.iv.name=Advanced Bundler IV
+gtadditions.machine.bundler.luv.name=Elite Bundler
+gtadditions.machine.bundler.luv.tooltip=Lapotron 3000
```
Comment (NotMyWing) [inline src/main/resources/assets/gtadditions/lang/en_us.lang:715]: Lapinotron

Comment (Exaxxion) [reply src/main/resources/assets/gtadditions/lang/en_us.lang:715]: Right, I misremembered the name we wanted to use. Will fix that.

**Comment by Exaxxion** [comment]
> I want to preface this by saying that @ALongStringOfNumbers asked me to review this PR.

Thank you for reviewing the PR. Constructive feedback is welcome from anyone, no need to justify it (and you're also far from a stranger). I appreciate you taking the time when I know you're already quite busy.

> Implementation-wise it is solid, I see no issues. However, personally I don't see why this should be in its own machine. Can these recipes not simply be added to the Packing Machine to save some machine bloat? I am of the mind that GregTech already has a few too many machines (2 Extractors, 2 Canners, Compressor and Bending Machine, Circuit and Regular Assembler in some versions, etc.), so personally I don't like another one-off machine.

We talked about this earlier on Discord and I can certainly understand where you're coming from. All I can really say is that I disagree about it being bloat. The packing machine is specialized for turning small piles of dusts into larger ones, and its counterpart the unpacking machine does the reverse. Mentally these processes strike me as quite disparate from, and requiring very different machines, than what would be going on in the Bundler.

Personally I think the way the Wiremill works and insulating with Assemblers is weird and this machine is something of a band-aid.

Wires are made very differently in reality: the metal is melted and rolled into rod which serves as the starting point for the wire. It might be fed directly through a hot drawing process, or cooled and transported to the next stage. The rod is then drawn (pulled) through dies made of diamond or tungsten carbide (depending on the needs of the material) until it is reduced to the desired thickness. Other steps like armoring, braiding, or stranding are performed, then insulation or jacketing is extruded onto the resulting wire.

So essentially, Wiremills should be a lot more like lubricated Extruders with variable sized dies to make the various thicknesses of solid wire, and insulating cables would actually take place in an Extruder. But... that's not what we have.

The current process of making small wires and kinda magically smooshing them together into a bigger gauge one is roughly like stranded cables I guess, though visually they look like a solid wire. So in this sense, the Bundler serves the purpose of stranding the wire. In reality it'd be a giant tube-shaped machine with a spinning drum.

**Review by ALongStringOfNumbers — COMMENTED** [review]
Looks good to me, just needs a config option for enabling the higher tier machines, to match with all the other machines added by SoG.

**Comment by Exaxxion** [comment]
Ah, good point. I'll add a config for that.

**Review by ALongStringOfNumbers — APPROVED** [review]

**Pull request merged**